libs/components/datetime/src/lib/modules/date-range-picker/fixtures/date-range-picker.component.fixture.ts

```diff
@@ -17,41 +17,45 @@ import { SkyDateRangeCalculatorId } from '../types/date-range-calculator-id';
   templateUrl: './date-range-picker.component.fixture.html',
 })
 export class DateRangePickerTestComponent implements OnInit, OnDestroy {
-  public get dateRange(): AbstractControl {
-    return this.reactiveForm.get('dateRange');
+  public get dateRange(): AbstractControl | undefined | null {
+    return this.reactiveForm?.get('dateRange');
   }
 
   @ViewChild('dateRangePicker', {
     read: SkyDateRangePickerComponent,
   })
-  public dateRangePicker: SkyDateRangePickerComponent;
+  public dateRangePicker!: SkyDateRangePickerComponent;
 
-  public calculatorIds: SkyDateRangeCalculatorId[];
-  public dateFormat: string;
+  public calculatorIds: SkyDateRangeCalculatorId[] | undefined;
+  public dateFormat: string | undefined;
   public disableReactiveOnInit = false;
   public endDateRequired = false;
   public initialDisabled = false;
-  public initialValue: SkyDateRangeCalculation;
-  public label: string;
+  public initialValue: SkyDateRangeCalculation | undefined;
+  public label: string | undefined;
   public numValueChangeNotifications = 0;
-  public reactiveForm: UntypedFormGroup;
+  public reactiveForm: UntypedFormGroup | undefined;
   public startDateRequired = false;
-  public templateDisable: boolean = undefined;
+  public templateDisable: boolean | undefined = undefined;
 
-  private ngUnsubscribe = new Subject<void>();
+  #ngUnsubscribe = new Subject<void>();
 
-  constructor(private formBuilder: UntypedFormBuilder) {}
+  #formBuilder: UntypedFormBuilder;
+
+  constructor(formBuilder: UntypedFormBuilder) {
+    this.#formBuilder = formBuilder;
+  }
 
   public ngOnInit(): void {
-    this.reactiveForm = this.formBuilder.group({
+    this.reactiveForm = this.#formBuilder.group({
```

Comment:
So the unit tests are currently written in a way that expects the ability to set an initial value between the construction and the first `detectChanges` being called. It would be possible to rewrite things but I think it would be more work than it would be worth for just a component fixture, thoughts?

Reply:
This sounds like a borderline onceability issue. The only reason it's not clear cut is because it's the "initial" value, so I can kind of rationalize requiring setting that before the component renders. However, that shouldn't keep us from being able to create `reactiveForm` in the constructor. We can create in the constructor and then set the value to the initial value in `ngOnInit()`.

Reply:
I took a look at this and think the couple of tests that were doing this weren't really necessary as other things cover what they were doing and they just complicated things. I decided to remove the tests and modified one other tests that could be done a slightly different way. Let me know if you have concerns.